crates/services/txpool_v2/src/pool_worker.rs

```diff
@@ -330,6 +343,9 @@ where
                     self.insert(tx, source);
                 }
             }
+            _ = pending_pool_expiration_check.tick() => {
+                self.pending_pool.expire_transactions(self.notification_sender.clone());
```

Comment:
It's not introduced in this PR, but I just noticed that internally, the `expire_transactions()` function sends out the `PoolNotification::ErrorInsertion` variant (in case of error).

Not sure if this is correct, but even if it isn't, it's not harmful. Leaving a comment for your consideration.